egui/src/widgets/text_edit.rs

```diff
@@ -182,9 +186,71 @@ impl<'t> TextEdit<'t> {
             enabled: true,
             desired_width: None,
             desired_height_rows: 4,
+            tab_as_spaces: false,
+            tab_moves_focus: true,
         }
     }
 
+    /// Registers if this widget will insert spaces instead of tab char
+    ///
+    /// ```rust, ignore
+    /// ui.add(egui::TextEdit::multiline(&mut self.multiline_text_input)
+    ///     .tab_as_spaces(true));
+    /// ```
+    pub fn tab_as_spaces(mut self, b: bool) -> Self {
```

Comment:
I'm not so sure about this feature, since it is only "half" supported. I think most people would expect that if this is set, then backspace would delete four spaces (it currently doesn't).

Personally I'd be happy to just support tabs, but if we should support spaces let's go all the way and make sure that you can also delete them four at a time!

Reply:
It is done